articles/becoming-a-flutter-developer/index.md

```diff
@@ -0,0 +1,179 @@
+
+For these reasons, the need for Flutter Developers will continue to increase. 
+
+
+### Table of Content
+- [Who is a Flutter Developer](#who-is-a-flutter-developer)
+- [Roles and Responsibilites of a Flutter Developer](#roles-and-responsibilites-of-a-flutter-developer)
+- [Flutter Developer Tools](#flutter-developer-tools)
+- [Flutter Developer Skills](#flutter-developer-skills)
+- [Mistakes to Avoid When Learning Flutter Development](#mistakes-to-avoid-when-learning-flutter-development)
+- [Getting a job as a Flutter Developer](#getting-a-job-as-a-flutter-developer)
+   - [Job Description Example](#job-description-example)
+   - [Flutter Developer Interview Questions](#flutter-developer-interview-questions)
+- [Further Reading](#further-reading)
+- [Conclusion ](#conclusion)
+
+
+### Who is a Flutter Developer
+
+A Flutter developer is a software developer who develops cross-platform applications. They use `Flutter Framework` to develop these applications using a single base code. 
+
+### Roles and Responsibilites of a Flutter Developer
+
+1. **To Code Applications.**
+
+It is the process of writing the code to develop software. Flutter Developers write this code on the Flutter framework.
+
+2. **TO Test Applications.**
+
+They test the functionality of the applications to make sure it does not crash. It also ensures the apps are running the well based on the system and user requirements.
+
+3. **To Deliver Applications.**
+
+It involves launching a complete application to the public. A mobile app is placed on `PlayStore` for Android applications or `AppleStore` for iOS applications. The feedback from the users is monitored and used to make changes and update the applications.
+
+
+### Flutter Developer Tools
+
+1. **Version Control Tools**
+
+These tools allow software developers to keep track of the changes they make on their projects. They also help for easy collaboration of a team of software developers while contributing to the same project. 
+
+Examples of version control tools:
+- [Git and Github](https://github.com/)
+- [Bitbucket](https://bitbucket.org/)
+
+2. **Flutter Framework**
+
+Flutter is a UI framework used to develop high performance and high-quality native applications for Android, iOs, the Web and desktop using a single code base. Dart is the primary programming language for Flutter Framework. It is a free and open-source framework. 
+
+Some features of Flutter that make it popular include 
+
+1. `Hot Reloading` which enables developers to see changes from the codebase in the UI. 
+2. `Hot Restart`. 
+3. `Present UI element` to create beautiful app designs for better `UI and UX`. 
+
+These UI elements are 
```

Comment:
Change `These UI elements are ...` to `These UI designs are....`  UI elements are mainly things such as Cards, buttons, etc. Also, explain more about material and Cupertino designs since they are part of Flutter's foundation